src/System.Globalization/tests/CompareInfo/CompareInfoTests.netstandard1.7.cs

```diff
@@ -297,17 +294,16 @@ public void SortKeyMiscTest()
             Assert.NotEqual(sk4.GetHashCode(), sk3.GetHashCode());
             Assert.NotEqual(sk4.KeyData, sk3.KeyData);
 
-            Assert.Equal(sk4, sk1);
-            Assert.Equal(sk4.GetHashCode(), sk1.GetHashCode());
-            Assert.Equal(sk4.KeyData, sk1.KeyData);
+            Assert.Equal(sk4, sk5);
+            Assert.Equal(sk4.GetHashCode(), sk5.GetHashCode());
+            Assert.Equal(sk4.KeyData, sk5.KeyData);
```

Comment:
Do these fail on Unix without this change? If so, do we know why? And if it's by design, should we have a Windows-only test to verify the original case?